lib/dgram.js

```diff
@@ -543,7 +563,7 @@ Socket.prototype.addMembership = function(multicastAddress,
   this._healthCheck();
 
   if (!multicastAddress) {
-    throw new Error('multicast address must be specified');
+    throw new errors.Error('ERR_MISSING_ARGS', 'multicastAaddress');
```

Comment:
Seems like we already use TypeError in other tests in this case as well. So TypeError is the way to go.